maro/cli/utils/web_terminal/requirements.txt

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Does need merge to maro/requirements.dev.txt?

Reply:
MARO did not import admin tool, seems admin tool's requirements are not need to be included in MARO.
add a  "maro admin req" command to install requirements for CLI admin tool